frontend/src/routes/[[lang=locale]]/(voters)/+layout.svelte

```diff
@@ -64,10 +57,10 @@
   setNavigationContext({close: closeDrawer});
 
   /**
-   * Create contexts for top application bar
+   * Create context for top application bar
    */
-  setTopBarContext();
-  setTopBarActionsConext();
+  initTopBarContext();
+
   setTopBarProgressContext({
```

Comment:
Okay! In terms of imports, what would you think of merging all these (top bar progress, top bar settings, navigation) contexts into the one and same layout context (and refactoring them so that the APIs are uniform)? Then we could just destructure the props when getting it?

Maybe the initialisation of the `max` and `current` stores could also be handled already when the context is created?